src/w3c/templates/sotd.js

```diff
@@ -177,33 +119,110 @@ function renderIsNoTrack(conf, opts) {
   `;
 }
 
-function renderImplementationReportURI(conf) {
-  const { implementationReportURI } = conf;
-  return html`<p>
-    Please see the Working Group's
-    <a href="${implementationReportURI}">implementation report</a>.
-  </p>`;
-}
-
-function renderNotRec({ anOrA, textStatus }) {
+function renderNotRec(conf) {
+  let statusExplanation = "";
+  let updatePolicy =
+    "This document may be updated, replaced or obsoleted at any time. It is inappropriate to cite this document as other than work in progress. ";
+  let reviewPolicy = "";
+  if (conf.specStatus === "CRD") {
+    statusExplanation =
+      "A Candidate Recommendation Draft integrates changes from the previous Candidate Recommendation that the Working Group intends to include in a subsequent Candidate Recommendation Snapshot.";
+    if (conf.lsMode) {
+      updatePolicy =
+        "This document is maintained and updated at any time. Some parts of this document are work in progress. ";
+    }
+  } else if (conf.specStatus === "CR") {
+    statusExplanation = html`A Candidate Recommendation Snapshot has received
+      <a href="https://www.w3.org/2020/Process-20200915/#dfn-wide-review"
+        >wide review</a
+      >
+      and is intended to gather
+      <a href="${conf.implementationReportURI}">implementation experience</a>.`;
+    updatePolicy = "";
+    if (conf.lsMode) {
+      reviewPolicy = `Comments are welcome at any time but most especially before ${conf.humanCREnd}.`;
+    } else {
```

Comment:
Maybe want to check `conf.humanCREnd` here? 

```suggestion
    } else if(conf.humanCREnd) {
```

Reply:
I'm not sure I understand - `humanCREnd` is needed in both conditions, and its absence is reported as an error in `w3c/headers.js`. What would this additional if brings?

Reply:
Ah, I might have misunderstood.